include/verilated.h

```diff
@@ -2493,6 +2494,65 @@ static inline void VL_ASSIGNSEL_WIIW(int obits, int lsb, WDataOutP owp, WDataInP
     _VL_INSERT_WW(obits, owp, rwp, lsb + obits - 1, lsb);
 }
 
+//======================================================================
+// Variable randomization (std::randomize, randomize method)
+
+inline IData VL_RANDOMIZE_VAR(CData& odata, int obits) VL_MT_SAFE {
+    odata = VL_RANDOM_Q(obits);
+    return 1;
+}
+inline IData VL_RANDOMIZE_VAR(SData& odata, int obits) VL_MT_SAFE {
+    odata = VL_RANDOM_Q(obits);
+    return 1;
+}
+inline IData VL_RANDOMIZE_VAR(IData& odata, int obits) VL_MT_SAFE {
+    odata = VL_RANDOM_Q(obits);
+    return 1;
+}
+inline IData VL_RANDOMIZE_VAR(QData& odata, int obits) VL_MT_SAFE {
+    odata = VL_RANDOM_Q(obits);
+    return 1;
+}
+
+inline IData VL_RANDOMIZE_VAR(WDataOutP odata, int lsb, int obits) VL_MT_SAFE {
+    VL_ASSIGNSEL_WIIQ(obits, lsb, odata, VL_RANDOM_Q(obits));
+    return 1;
+}
+
+inline IData VL_RANDOMIZE_ENUM_VAR(const CData* values, int valCnt, CData& odata) VL_MT_SAFE {
+    odata = values[vl_rand64() % valCnt];
+    return 1;
```

Comment:
Also removing these will allow verilator to optimize the %'s to shifts and potentially make other improvements.

(Randomize can be a hot function, unlike the reset code you probably copied this from)